Build-your-first-dApp.md

```diff
@@ -128,36 +146,44 @@ OPTIONAL: Inside the `<head>` tag, add some styles to make it look nicer
 
 Now it's time to create a Solidity smart contract.
 
-1. You can use any editor you like to make the contract. For this tutorial we recommend the online IDE [Remix](https://remix.ethereum.org)
+1. You can use any editor you like to make the contract. For this part of the tutorial we recommend the online IDE [Remix](https://remix.ethereum.org)
 2. Go to [Remix](https://remix.ethereum.org)
 3. Check out the "Solidity Compiler", and "Deploy and Run Transactions" tabs. If they are not present, enable them in the plugin manager
 4. Create a new solidity file in remix, named `mood.sol`
 5. Write the contract
 
    - Specify the solidity version and add a license
+    
+    <br>
```

Comment:
Here and the places below as well.

Reply:
I've replaced them with <p>'s ( since we only needed some line spacing ) and it seems to work fine, can you check it out as well please?